src/services/RedocNormalizedOptions.ts

```diff
@@ -110,6 +111,16 @@ export class RedocNormalizedOptions {
     return value;
   }
 
+  private static normalizeSampleCollapseLevel(level?: number | string | 'all'): number {
```

Comment:
You forgot to rename this to `normalizeSampleExpandLevel`.

```suggestion
  private static normalizeSampleExpandLevel(level?: number | string | 'all'): number {
```

Reply:
Thanks, fixed